packages/frontend/src/components/FlowRow/DemoFlowModal.tsx

```diff
@@ -11,15 +11,26 @@ import {
 } from '@chakra-ui/react'
 import { Badge, BadgeLeftIcon, Button } from '@opengovsg/design-system-react'
 import demoModalImg from 'assets/demo-modal.png'
+import {
+  DEMO_VIDEOS_MAP,
+  FORMSG_POSTMAN_TEMPLATE,
+} from 'helpers/flow-templates'
 
 import DemoVideoModalContent from './DemoVideoModalContent'
 
 interface DemoFlowModalProps {
   onClose: () => void
+  isPreCreated?: boolean
+  demoVideoId?: string
 }
 
 export default function DemoFlowModal(props: DemoFlowModalProps): ReactElement {
-  const { onClose } = props
+  const { onClose, isPreCreated, demoVideoId } = props
+  // fallback to default demo video to display for GGWP v1
+  const { url, title } = demoVideoId
+    ? DEMO_VIDEOS_MAP[demoVideoId]
+    : DEMO_VIDEOS_MAP[FORMSG_POSTMAN_TEMPLATE]
```

Comment:
u can either expand the props to { onClose, isPreCreated, demoVideoId = FORMSG_POSTMAN_TEMPLATE }

or

const { url, title } = DEMO_VIDEOS_MAP[demoVideoId ?? FORMSG_POSTMAN_TEMPLATE]